tests/common.go

```diff
@@ -5131,3 +5131,140 @@ func GetPoolIDsFromVolName(volName string) ([]string, error) {
 	}
 	return poolUuids, err
 }
+
+// GetNodeByNodeName Get Node Details from Node Name Provided
+func GetNodeDetailsByNodeName(nodeName string) (*node.Node, error) {
+	Nodes, err := GetStorageNodes()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, each := range Nodes {
+		if each.Name == nodeName {
+			return &each, nil
+		}
+	}
+	return nil, err
+}
+
+// GetPoolIOPriorityStatus Get Pool IO Priority status from PoolUUID
+func GetPoolIOPriorityStatus(poolUUID string) (string, error) {
+	var PoolIOPriority string
+	PoolIOPriority = ""
+	Pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+	log.FailOnError(err, fmt.Sprintf("Listing storage pools failed"))
+	for _, eachPool := range Pools {
+		if eachPool.Uuid == poolUUID {
+			PoolIOPriority = eachPool.Labels["iopriority"]
```

Comment:
can you move this to portworx.go and take the label as the key, so in future we can use same function to get another label also

Reply:
done